src/pyenphase/envoy.py

```diff
@@ -177,7 +179,7 @@ async def _request(
         url = self.auth.get_endpoint_url(endpoint)
 
         if data:
-            _LOGGER.debug("Sending POST to %s with data %s", url, data)
+            _LOGGER.debug("Sending POST to %s with data %s", url, orjson.dumps(data))
```

Comment:
Guard this with is isEnabledFor since the dumps will happen even if debug logging isn't enabled